owncloudData/src/main/java/com/owncloud/android/data/files/db/FileDao.kt

```diff
@@ -279,6 +280,7 @@ interface FileDao {
                 permissions = null,
                 treeEtag = "",
                 availableOfflineStatus = NOT_AVAILABLE_OFFLINE.ordinal,
+                lastUsage = null
```

Comment:
This is not needed, null is already the default value for `lastUsage` in the constructor, same as `etagInConflict` for example

Reply:
Done